### ISSUE TITLE
What does Glue do with WCS and reprojection in Jupyter notebook?

### ISSUE THREAD

**Issue by pllim** [spacetelescope/jdaviz#25]
Duplicate of glue-viz/glue-jupyter#112 especially created for JIRA.

p.s. @astrofrog is doing the real work. I am just being assigned in name. 😬

**Comment by stscijgbot** [comment]
This ticket is now being tracked at [DATJPP-113](https://jira.stsci.edu/browse/DATJPP-113)

**Comment by stscijgbot** [comment]
This ticket is now being tracked at [DATJPP-120](https://jira.stsci.edu/browse/DATJPP-120)

**Comment by SaOgaz** [comment]
@pllim or @astrofrog, are we waiting on more information for this ticket still?

**Comment by pllim** [comment]
I dunno. Turns out @ibusko got something else to work outside of `glue-jupyter`. Does @eteq and @camipacifici will want to pursue this?

**Comment by nmearl** [comment]
It's not clear what this is supposed to address. Feel free to open with new information, or any new associated issues.

**Comment by pllim** [comment]
I don't remember anymore either.
